resources/js/src/app/services/UrlService.js

```diff
@@ -1,6 +1,6 @@
 import $ from "jquery";
-import {isNullOrUndefined}from "../helper/utils";
```

Comment:
Maybe replace it with `import {isDefined}from "../helper/utils";` ...

Reply:
Done